### PR TITLE
[CELEBORN-751] Rename remain rss related class name and filenames etc...

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Rename remain rss related class name and filenames etc...

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture @RexXiong @kerwin-zk @FMX @pan3793

File: client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java [hunk 1]
```diff
@@ -51,13 +51,13 @@ public class RemoteShuffleMaster implements ShuffleMaster<RemoteShuffleDescripto
           ThreadUtils.createFactoryWithDefaultExceptionHandler(
               "remote-shuffle-master-executor", LOG));
   private final ResultPartitionAdapter resultPartitionDelegation;
-  private final long rssMetaServiceTimestamp;
+  private final long metaServiceTimestamp;
```
Comment (pan3793) [inline client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java:54]: meta service is the alias of master? @waitinfuture

Comment (waitinfuture) [reply client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java:54]: meta service is lifecyclemanager

File: client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/RssShuffleManager.java
Comment (FMX) [inline client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/RssShuffleManager.java:1]: Remaining unchanged for compatibility reasons?

Comment (AngersZhuuuu) [reply client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/RssShuffleManager.java:1]: Yea, need discuss

File: common/src/main/java/org/apache/celeborn/common/protocol/RpcNameConstants.java [hunk 2]
```diff
@@ -31,6 +31,6 @@ public class RpcNameConstants {
   public static String WORKER_EP = "WorkerEndpoint";
 
   // For Driver(RssShuffleManager)
-  public static String RSS_METASERVICE_EP = "RssMetaServiceEndpoint";
-  public static String RSS_METASERVICE_SYS = "RssMetaSys";
+  public static String METASERVICE_EP = "MetaServiceEndpoint";
```
Comment (FMX) [inline common/src/main/java/org/apache/celeborn/common/protocol/RpcNameConstants.java:34]: It's clear that our meta is LifecycleManager and it's engine independent. Why not change it to LIFECYCLEMANAGER_EP? There is no need to add a new concept here.

Comment (AngersZhuuuu) [reply common/src/main/java/org/apache/celeborn/common/protocol/RpcNameConstants.java]: DONE

**Comment by AngersZhuuuu** [comment]
ping @FMX @waitinfuture @pan3793 @RexXiong Could you take a look

**Review by pan3793 — APPROVED** [review]

File: docs/configuration/worker.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ license: |
 | celeborn.worker.storage.dirs | &lt;undefined&gt; | Directory list to store shuffle data. It's recommended to configure one directory on each disk. Storage size limit can be set for each directory. For the sake of performance, there should be no more than 2 flush threads on the same disk partition if you are using HDD, and should be 8 or more flush threads on the same disk partition if you are using SSD. For example: `dir1[:capacity=][:disktype=][:flushthread=],dir2[:capacity=][:disktype=][:flushthread=]` | 0.2.0 | 
 | celeborn.worker.storage.disk.reserve.size | 5G | Celeborn worker reserved space for each disk. | 0.3.0 | 
 | celeborn.worker.storage.hdfs.dir | &lt;undefined&gt; | HDFS dir configuration for Celeborn to access HDFS. | 0.2.0 | 
-| celeborn.worker.storage.workingDir | rss-worker/shuffle_data | Worker's working dir path name. | 0.3.0 | 
+| celeborn.worker.storage.workingDir | celeborn-worker/shuffle_data | Worker's working dir path name. | 0.3.0 |
```
Comment (pan3793) [inline docs/configuration/worker.md:96]: this may need to upgrade migration guide

**Comment by waitinfuture** [comment]
Thanks @AngersZhuuuu for this Refactor! Seems we lack UTS for columnar shuffle/Codegen @kerwin-zk

**Pull request closed without merging**